### ISSUE TITLE
buy populated PCBs?

### ISSUE THREAD

**Issue by hackrid** [carrotIndustries/pluto#1]
Hi,

is there a way to buy populated PCBs ?

BR

**Comment by luftek** [comment]
Probably not relevant any more? I did order new batch? #11

**Comment by vasev01** [comment]
I’m also interested in buying one populated PCB.

**Comment by carrotIndustries** [comment]
Don't hold your breath, but I might to a second board cut in the next few months. Stay tuned!

**Comment by zigastegu** [comment]
Compass is hard to get a hold off. It is a deprecated part.

**Comment by carrotIndustries** [comment]
Yeah, that's why I was planning on removing the magnetometer in the next revision. It's been more of a gimmick anyhow.

**Comment by vasev01** [comment]
@carrotIndustries A year has passed since you announced a second board in few months. I'm wearing the original casio all the time hoping to be able to program it. Any positive news?

**Comment by slavaaaaaaaaaa** [comment]
I, too, am still holding my breath.

**Comment by carrotIndustries** [comment]
I'm really sorry, but I'm too busy with other projects to get a new pluto revision out the door.

Anyhow, this project is open source hardware, so feel free to organize make a new rev on your own. 

If I'd do it, I'd change these things:
 - Drop compass (EOL and not that useful after all) and IR sensor (no firmware support)
 - Panelize the PCB to drive cost down
 - Make the Vcc capacitor 1µF

If someone really wants to pull this off, I can do design support and review on a best-effort basis.

Good luck!

**Comment by wannessels** [comment]
Could you have a look at my [pull request](https://github.com/carrotIndustries/pluto/pull/6)? I'd like to order a batch of boards

**Comment by tandy-1000** [comment]
im also interested in this

**Comment by wannessels** [comment]
I'll have a go at ordering new PCB's.

Anyone who's interested, can you tell me what and how many you're interested in?
- Bare PCB's
- PCB + components, unsoldered. You will need some sort of MSP430 devkit to flash the mcu
- PCB + components, soldered and flashed. You will still need to desolder and reuse the inductor from a donor F-91W

**Comment by vasev01** [comment]
@wannessels, the bare PCB is not a problem nowadays just drop a zip file with gerbers and fill in your address and you are done. However, soldering small SMD components, especially those VQFNs and BGAs, is the problem for many of us.
So, my choice would be PCB with soldered components (at least CPU).

Thanks,
Eugene.

**Comment by lpasselin** [comment]
How much would that be total for each popupated PCB including parts?

Edit: 2-3 boards here flashed ideally

**Comment by slavaaaaaaaaaa** [comment]
Depending on pricing, I'd go for somewhere between two and five populated boards.

**Comment by tandy-1000** [comment]
id be interested in one populated board
